File: YourClassroom/app/src/main/java/android/example/yourclassroom/MainActivity.java
package android.example.yourclassroom;

import android.content.Intent;
import android.example.yourclassroom.view.exercise.ListExerciseActivity;
import android.os.Bundle;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_shows_class);

        String idClass = "-ONFpWG_4Kch5GD26f7x";
        String idUser = "1";
        String idTeacher = "1";

        Intent ListExerciseIntent = new Intent(this, ListExerciseActivity.class);
        ListExerciseIntent.putExtra("idClass", idClass);
        ListExerciseIntent.putExtra("idUser", idUser);
        ListExerciseIntent.putExtra("idTeacher", idTeacher);
        startActivity(ListExerciseIntent);
    }
}

File: YourClassroom/app/src/main/java/android/example/yourclassroom/MainActivity.kt
package android.example.yourclassroom

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var text: TextView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val sharedPreferences = getSharedPreferences("MyAppPrefs", Context.MODE_PRIVATE)
        val isLoggedIn = sharedPreferences.getBoolean("isLoggedIn", false)

        if (!isLoggedIn) {
            val intent = Intent(this, ComposeActivity::class.java)
            intent.putExtra("startDestination", "login")
            startActivity(intent)
            finish()
        }

        val getSharedPreferences = getSharedPreferences("MyAppPrefs", Context.MODE_PRIVATE)
        val displayName = getSharedPreferences.getString("displayName", "")
        text = findViewById(R.id.welcome)
        text.text = "Xin chao, $displayName!"

    }
}